.github/workflows/check-release.yaml

```diff
@@ -33,6 +33,7 @@ jobs:
             
       - name: R CMD CHECK inside r
         run : |
+            remotes::install_version("mlr3viz", version = "0.5.10")
```

Comment:
We have to add `install.packages("remotes")`  in Dockerfile before `remotes::install_version("mlr3vix")`